### PR TITLE
Refactor seed

### PR DESCRIPTION
**Pull request by NapoleonDLP**
- Completes seeding script for Cassandra

### CODE REVIEW

File: database/cassandra/seedCassandra.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const { generateProduct } = require('../seedHelpers.js');
+const { client } = require('./cassandraConnection.js');
+
+const seedCasandra = async () => {
+  const keyspaceQuery = "CREATE KEYSPACE IF NOT EXISTS related_products WITH REPLICATION = { 'class': 'SimpleStrategy' , 'replication_factor': 3 }";
+  const truncateTableQuery = 'TRUNCATE related_products.products';
+  const createTableQuery = 'CREATE TABLE IF NOT EXISTS related_products.products (id int PRIMARY KEY, name text, price text, styles text, style_thumbnails text)';
+
+  client.execute(keyspaceQuery)
+    .then(() => client.execute(keyspaceQuery))
```
Comment (merlinv12) [inline database/cassandra/seedCassandra.js:10]: looks like you have keyspace query twice?

File: newrelic.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['garrett-related-products'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'a32f7cb9743197698370ee339fc94d0e430ccff3',
```
Comment (merlinv12) [inline newrelic.js:16]: Be careful with your keys on github!  You can setup environmental variables so that this isn't displayed.

**Review by merlinv12 — COMMENTED** [review]
Looks good other than the key. I'll talk to you before I merge to see what you want to do.

File: newrelic.js (hunk 2 above)
Comment (NapoleonDLP) [reply newrelic.js:16]: Ah yes. Thanks. Ill set that up.

**Pull request merged**